src/ms-store-badge.ts

```diff
@@ -288,50 +344,50 @@ class MSStoreBadge extends HTMLElement {
-      { name: "Croatian", code: "hr", imageSmall: { fileName: "Croatian_S.png" }, imageLarge: { fileName: "Croatian_L.png" } },
-      { name: "Czech", code: "cs", imageSmall: { fileName: "Czech_S.png" }, imageLarge: { fileName: "Czech_L.png" } },
-      { name: "Danish", code: "da", imageSmall: { fileName: "Danish_S.png" }, imageLarge: { fileName: "Danish_L.png" } },
-      { name: "Dutch", code: "nl", imageSmall: { fileName: "Dutch_S.png" }, imageLarge: { fileName: "Dutch_L.png" } },
+      { name: "Arabic", code: "ar", imageSmall: { fileName: "Arabic_S.png" }, imageSmallLight: {fileName: "English_SL.jpg"}, imageLarge: { fileName: "Arabic_L.png" }, imageLargeLight: { fileName: "English_LL.jpg"} },
+      { name: "Bosnian", code: "be", imageSmall: { fileName: "Bosnian_S.png" }, imageSmallLight: {fileName: "English_SL.jpg"}, imageLarge: { fileName: "Bosnian_L.png" }, imageLargeLight: { fileName: "English_LL.jpg"} },
+      { name: "Bengali", code: "bn", imageSmall: { fileName: "Bengali_S.png" }, imageSmallLight: {fileName: "English_SL.jpg"}, imageLarge: { fileName: "Bengali_L.png" }, imageLargeLight: { fileName: "English_LL.jpg"} },
+      { name: "Bosnian", code: "bs", imageSmall: { fileName: "Bosnian_S.png" }, imageSmallLight: {fileName: "English_SL.jpg"}, imageLarge: { fileName: "Bosnian_L.png" }, imageLargeLight: { fileName: "English_LL.jpg"} },
+      { name: "Bulgarian", code: "bg", imageSmall: { fileName: "Bulgarian_S.png" }, imageSmallLight: {fileName: "English_SL.jpg"}, imageLarge: { fileName: "Bulgarian_L.png" }, imageLargeLight: { fileName: "English_LL.jpg"} },
+      { name: "Chinese (Simplified)", code: "zh-cn", imageSmall: { fileName: "Chinese_Simplified_S.png" }, imageSmallLight: {fileName: "English_SL.jpg"}, imageLarge: { fileName: "Chinese_Simplified_L.png" }, imageLargeLight: { fileName: "English_LL.jpg"} },
+      { name: "Chinese (Traditional)", code: "zh-tw", imageSmall: { fileName: "Chinese_Traditional_S.png" }, imageSmallLight: {fileName: "English_SL.jpg"}, imageLarge: { fileName: "Chinese_Traditional_L.png" }, imageLargeLight: { fileName: "English_LL.jpg"} },
+      { name: "Croatian", code: "hr", imageSmall: { fileName: "Croatian_S.png" }, imageSmallLight: {fileName: "English_SL.jpg"}, imageLarge: { fileName: "Croatian_L.png" }, imageLargeLight: { fileName: "English_LL.jpg"} },
+      { name: "Czech", code: "cs", imageSmall: { fileName: "Czech_S.png" }, imageSmallLight: {fileName: "English_SL.jpg"}, imageLarge: { fileName: "Czech_L.png" }, imageLargeLight: { fileName: "English_LL.jpg"} },
+      { name: "Danish", code: "da", imageSmall: { fileName: "Danish_S.png" }, imageSmallLight: {fileName: "English_SL.jpg"}, imageLarge: { fileName: "Danish_L.png" }, imageLargeLight: { fileName: "English_LL.jpg"} },
+      { name: "Dutch", code: "nl", imageSmall: { fileName: "Dutch_S.png" }, imageSmallLight: {fileName: "English_SL.jpg"}, imageLarge: { fileName: "Dutch_L.png" }, imageLargeLight: { fileName: "English_LL.jpg"} },
```

Comment:
If we're deploying to production, let's make the light mode images be the same as the dark mode images until we have the new designs from Andy. You can delete the test images.